packages/clay-card/src/AspectRatio.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import Context from './Context';
+
+import classNames from 'classnames';
+
+import React from 'react';
+
+interface CardAspectRatioProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Aspect ratio to be applied on the background image of the card.
+	 */
+	backgroundImageAspectRatio?: BackgroundImageAspectRatioType;
+
+	/**
+	 * Contrains an image for a given Aspect Ratio.
+	 */
+	containerAspectRatio?: ContainerAspectRatioType;
+
+	children:
```

Comment:
Is it necessary to define children here? I would assume this would be extended from HTMLDivElement

Reply:
Yes, Isn't necessary. Removing!